### PR TITLE
You can no longer dupe durathread

### PR DESCRIPTION
**Pull request by redmoogle**
# Document the changes in your pull request

You could merge it with cloth lol

# Changelog

:cl:   
bugfix: Durathread can no longer be duped with cloth
/:cl:

### CODE REVIEW

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -416,6 +416,7 @@ GLOBAL_LIST_INIT(silk_recipes, list ( \
 	merge_type = /obj/item/stack/sheet/cloth/durathread
 	drop_sound = 'sound/items/handling/cloth_drop.ogg'
 	pickup_sound =  'sound/items/handling/cloth_pickup.ogg'
+	merge_type = /obj/item/stack/sheet/cloth/durathread
```
Comment (tattax) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: ?

Comment (redmoogle) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: Merge type allows for it to merge its types and subtypes

Comment (tattax) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: there's two of those

Comment (redmoogle) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: > there's two of those

Whoops ill go fix it properly then in a bit

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**